.github/workflows/ci-tests-app.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
     - name: Set min. dependencies
       if: ${{ matrix.requires == 'oldest' }}
-      run: python .actions/assistant.py replace_oldest_ver
+      run: python .actions/assistant.py replace_oldest_ver --requirement_fnames="[requirements/app/base.txt, requirements/app/ui.txt, requirements/app/cloud.txt, requirements/app/test.txt]"
```

Comment:
```suggestion
      run: python .actions/assistant.py replace_oldest_ver
```

Reply:
this is the only way I got the app example tests to work